packages/ipfs-http-client/src/pubsub/subscribe.js

```diff
@@ -45,6 +48,9 @@ module.exports = configure((api, options) => {
 
     clearTimeout(ffWorkaround)
 
+    // Note: It's interesting that subscribe
+    // keeps this ndjson(tranformation(res)) pattern although
+    // that's now long from other IPFS methods
     readMessages(ndjson(streamToAsyncIterator(res)), {
```

Comment:
@hugomrdias For parallelism, would it make sense to get rid of this `ndjson(transformer(res))` pattern here too since it was removed for other IPFS methods in the `ky` removal PR?